packages/replay/src/coreHandlers/handleScope.ts

```diff
@@ -44,22 +47,24 @@ export function handleScope(scope: Scope): Breadcrumb | null {
   _LAST_BREADCRUMB = newBreadcrumb;
 
   if (
-    newBreadcrumb.category &&
-    (['fetch', 'xhr', 'sentry.event', 'sentry.transaction'].includes(newBreadcrumb.category) ||
-      newBreadcrumb.category.startsWith('ui.'))
+    typeof newBreadcrumb.category !== 'string' ||
```

Comment:
do we need this? seems like more bytes... what about making this a type predicate function:

```ts
function isBreadcrumbWithCategory(breadcrumb: Breadcrumb): breadcrumb is BreadcrumbWithCategory {
  return !!breadcrumb.category;
}

// later
if (!isBreadcrumbWithCategory(newBreadcrumb) || [...].includes(...) {
  return null;
}
// also no need to cast here anymore
```